x/farming/spec/03_state_transitions.md

```diff
@@ -31,10 +31,10 @@ const (
   - Each `farmingPlan` has its own `startTime` and `endTime`
 - Distribution Method
   - `FixedAmountPlan`
-    - fixed amount of coins are distributed for each `epochDays`
+    - fixed amount of coins are distributed for each `NextEpochDays`
```

Comment:
I'm afraid if this sentence is correct, we could just say "fixed amount of coins are distributed for each `CurrentEpochDays`

Reply:
`03_state_transitions.md` spec docs has some rooms for improvement. Let's use your sentence and handle it with another PR to improve spec documentation.